### ISSUE TITLE
Debug mode

### ISSUE THREAD

**Issue by captyoungkyu** [djfdyuruiry/swagger2-postman-generator#1]
What a wonderful lib!!!

This is an amazing library and thank you for making it.
The only bad thing is 'console.log'. What about creating a way to remove the logs?

For example,

```javascript
Swagger2PostmanGenerator.convertSwagger({ debug: false }).fromSpec(swaggerSpec);
```

what do you think?

**Comment by djfdyuruiry** [comment]
Hi @captyoungkyu 

Good catch, I think this is an excellent idea, I have implemented this change in v2.0.7 of the NPM package: see [github commit](https://github.com/djfdyuruiry/swagger2-postman-generator/commit/3e22d589661ffa832b51b763e7fb0710243879a5) & [npm package](https://www.npmjs.com/package/swagger2-postman-generator)

I am glad someone is using this and finding it useful 👍

**Comment by captyoungkyu** [comment]
Thank you very much!

**Comment by captyoungkyu** [comment]
```
node_modules/swagger2-to-object/swagger2-to-object.js:358

    if (options && options.debug) {
        ^
ReferenceError: options is not defined
```

Very sadly, 2.0.7 version does not work with the above error.

**Comment by msnoddy** [comment]
@captyoungkyu fixed in v2.0.8 and above, enjoy
